src/main/java/com/exasol/releasedroid/usecases/release/LocalReleaseInteractor.java

```diff
@@ -0,0 +1,19 @@
+package com.exasol.releasedroid.usecases.release;
+
+import java.util.List;
+
+import com.exasol.errorreporting.ExaError;
+import com.exasol.releasedroid.usecases.UserInput;
+import com.exasol.releasedroid.usecases.report.Report;
+
+/**
+ * This class prevents a release on local repository. Currently we only release on the GitHub.
+ */
+public class LocalReleaseInteractor implements ReleaseUseCase {
+    @Override
+    public List<Report> release(final UserInput userInput) {
```

Comment:
This class should not exist, and instead we should validate in the `UserInputParser` that, if the `--local` parameter is set, then the `release` goal is not allowed.

Same should go with all mutually exclusive combinations, ie, `--local` and `--branch`

Reply:
But I still need to provide a ReleaseUseCase object (in Runner where I create all the objects). I don't want to provide the second one we have, because I don't want to ever use it with a local setup. I don't want to provide null either, because it can cause a null pointer. I think this empty class is better than null. Any other solutions?